packages/core/state/metadata/selectors.ts

```diff
@@ -1,27 +1,5 @@
-import { createSelector } from "reselect";
-
 import { State } from "../";
-import Annotation, { AnnotationName } from "../../entity/Annotation";
 
 // BASIC SELECTORS
 export const getAnnotations = (state: State) => state.metadata.annotations;
 export const getDataSources = (state: State) => state.metadata.dataSources;
-
-// COMPOSED SELECTORS
-export const getSortedAnnotations = createSelector(getAnnotations, (annotations: Annotation[]) => {
```

Comment:
Looks like since you didn't end up doing the sorting of the recent annotation in this selector you can move it back to this file

Reply:
resolved in [0faf718](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/110/commits/0faf718ed4318494cd1d819ddd75348c06910a2c)